cranelift/filetests/filetests/runtests/bint.clif

```diff
@@ -0,0 +1,20 @@
+test run
+target aarch64
+target x86_64 machinst
+
+function %bint_b8_i16() -> i16 {
+block0:
+  v0 = bconst.b8 true
```

Comment:
For the sake of completeness, shouldn't we also have tests for `false`?

Reply:
Done!